Google-Summer-of-Code-2024.md

```diff
@@ -650,15 +652,13 @@ Link to PRD: https://docs.google.com/document/d/1r9IEQ5z_t-eu9XAWN3eRA7iKdKuYsOQ
 Note: Ability to [write Beam jobs](https://github.com/oppia/oppia/wiki/Apache-Beam-Jobs) is a plus, but is not strictly required.
 
 **Suggested Milestones:**
-- **Milestone 1**: Add a classification level for each skill as trivial/simple/regular, and allow the curriculum admin / topic manager to update this classification in the skill editor (with a default to regular). Include help text for them to explain what each classification means.
+- **Milestone 1**: Track each user’s total points (this should be updated in the backend each time a learner answers a question correctly, regardless of whether they eventually complete the practice session or not), and show these totals in their learner dashboard. The tracking functionality should be hidden behind a `TRACK_PRACTICE_QUESTION_POINTS` feature flag, and the display functionality should be hidden behind a `SHOW_PRACTICE_QUESTION_POINTS` feature flag.
```

Comment:
> and show these totals

and show these points?

Reply:
Should probably be "and show this total". Will fix, thanks!

Reply:
Done